tests/ReactTestApp/ios/Podfile.lock

```diff
@@ -272,7 +272,7 @@ PODS:
     - React-perflogger (= 0.64.2)
   - ReactNativeExceptionHandler (2.10.10):
     - React-Core
-  - RealmJS (10.5.0):
+  - RealmJS (10.6.0):
```

Comment:
It's fine this is updated here, but I'm in doubt if we even want / need to keep this checked in? These versions and hashes are derived from the NPM packages, which already have the `package-lock.json` to enforce a deterministic install.

Reply:
Sure, makes sense.  I've always committed it before, but it really doesn't "need" to be there.